Ports/ffmpeg/package.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=ffmpeg
+version=4.4
+useconfigure=true
+files="https://ffmpeg.org/releases/ffmpeg-4.4.tar.gz ffmpeg-4.4.tar.gz a08992a9f62cf9877582565544e8e799"
+auth_type="md5"
+makeopts="-j$(nproc)"
+installopts="INSTALL_TOP=${SERENITY_INSTALL_ROOT}/usr/local"
+configopts="SRC_PATH=."
+
+configure() {
+    run ./configure \
+		--arch="x86_32" \
+		--cc="i686-pc-serenity-gcc -std=gnu99" \
+		--cxx="i686-pc-serenity-g++ -std=gnu99" \
+		--disable-network \
+		--enable-cross-compile
```

Comment:
- Please don't mix tabs and spaces (spaces only)
- Please don't hardcode `i686` (use `${CC}` and `${CXX}` instead of `i686-pc-serenity-gcc` and `i686-pc-serenity-g++`